name.abuchen.portfolio/src/name/abuchen/portfolio/util/TradeCalendarManager.java

```diff
@@ -141,6 +152,27 @@ public class TradeCalendarManager
         tc.add(fixed(STATE_FUNERAL, Month.DECEMBER, 5).onlyIn(2018)); // funeral of former president Bush Sr.
         CACHE.put(tc.getCode(), tc);
 
+        // see https://www.bolsadesantiago.com/mercado_horarios_feriados
+        tc = new TradeCalendar("sgo", Messages.LabelTradeCalendarSGO, STANDARD_WEEKEND); //$NON-NLS-1$
+        tc.add(fixed(NEW_YEAR, Month.JANUARY, 1));
+        tc.add(easter(GOOD_FRIDAY, -2));
+        tc.add(fixed(HOLY_SATURDAY, Month.MARCH, 30));
```

Comment:
First, Saturdays and Sundays are non-trading days by default (because you added the STANDARD_WEEKEND).

Second, you cannot use a ```fixed``` date here, because the holiday is not every year on 30 March. It is in 2024, but not in 2025.

I will remove this.